index.html

```diff
@@ -475,7 +475,7 @@
       if (evt.data == "say hello")
         session.send("hello");
     }
```

Comment:
Could you update the PR to add the missing semicolon (;) on L477 so we fix that in the same PR, thanks!